src/globals/theme/_functions.scss

```diff
@@ -10,7 +10,6 @@
 /// @param {Map<String, Color>} $theme The theme to remove the type tokens from.
 /// @returns {Map<String, Color>} The transformed theme.
 /// @access private
-/// @deprecated
```

Comment:
Is this no longer deprecated?

Reply:
This is a private function but shouldn't be marked as deprecated because it's still in use